download.md

```diff
@@ -365,6 +365,13 @@ To improve the speed and load time of your site, build Prebid.js for only the he
     </div>
   </div>
 
+  <div class="col-md-4">
+    <div class="checkbox">
+      <label>
+        <input type="checkbox" bidderCode="hiromedia" class="bidder-check-box"> HIRO Media
```

Comment:
Thanks, I wasn't sure about this one. Rebased. 

Now based on previous code I saw.